extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -131,6 +134,16 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	async createBook(bookPath: string, contentPath: string): Promise<void> {
+		bookPath = path.normalize(bookPath);
+		contentPath = path.normalize(contentPath);
+		await this.bookTocManager.createBook(bookPath, contentPath);
+	}
+
+	async editBook(book: BookTreeItem, section: BookTreeItem, sectionParent: BookTreeItem): Promise<void> {
```

Comment:
sectionParent isn't being used?

Reply:
Oh that's right! I forgot to remove it. Thank you!